packages/react-storage/src/components/StorageBrowser/controls/hooks/__tests__/useDropZone.spec.tsx

```diff
@@ -0,0 +1,31 @@
+import { useControlsContext } from '../../../controls/context';
+import { useDropZone } from '../useDropZone';
+
+jest.mock('../../../controls/context');
+
+describe('useDropZone', () => {
+  // assert mocks
+  const mockUseControlsContext = useControlsContext as jest.Mock;
+
+  afterEach(() => {
+    mockUseControlsContext.mockReset();
```

Comment:
It's not really required either way given that this only has one test but since I'm mocking return value here, reset would guarantee that a future test would not just consume the same mocked return value without explicitly refactoring the tests to do so